yarn.lock

```diff
@@ -27519,6 +27519,11 @@ zod@^3.19.0:
   resolved "https://registry.npmjs.org/zod/-/zod-3.19.0.tgz#a9303fcaed0950a3dc57bf35cc11f6282d4f52f2"
   integrity sha512-Yw0qvUsCNGBe5YacikdMt5gVYeUuaEFVDIHKMfElrGSaBhwR3CQK6vOzgfAJOjTdGIhEeoaj8GtT+NDZrepZbw==
 
+zod@^3.19.1:
```

Comment:
can we use the same version of zod everywhere? lets hardcode it to 3.19.1

Reply:
[`3d5f0c7` (#3542)](https://github.com/dendronhq/dendron/pull/3542/commits/3d5f0c77d0cc2f0f6f27340bc9035ffde40a6a7b)
[`28a529e` (#3542)](https://github.com/dendronhq/dendron/pull/3542/commits/28a529efcf53f8c2cc00318cccacbe65d154faf0)